README.md

```diff
@@ -436,6 +467,29 @@ segment, as well:
 # Output time, date, and a symbol from the "Awesome Powerline Font" set
 POWERLEVEL9K_TIME_FORMAT="%D{%H:%M:%S \uE868  %d.%m.%y}"
 ```
+##### user
+
+The `user` segment (user string) will print the username. You can customize the `user` segment.
+For example, you can modify the COLOURS for the two states - DEFAULT and ROOT, by setting
+```
+POWERLEVEL9K_USER_DEFAULT_BACKGROUND="green"
+POWERLEVEL9K_USER_DEFAULT_FOREGROUND="white"
+POWERLEVEL9K_USER_ROOT_BACKGROUND="red"
+POWERLEVEL9K_USER_ROOT_FOREGROUND="yellow"
+```
+
+You can also override the icons by setting
+```
+POWERLEVEL9K_USER_ICON="\uF415" # 
+POWERLEVEL9K_ROOT_ICON="#"
+```
+
+| Variable | Default Value | Description |
+|----------|---------------|-------------|
+|`DEFAULT_USER`|None|Username to consider a "default context" (you can also set `$USER`).|
+|`POWERLEVEL9K_ALWAYS_SHOW_USER`|false|Always show the username, but conditionalize the hostname.|
```

Comment:
It seems like the end of this description doesn't make sense, "...but conditionalize the hostname". I'm guessing this is just a copy/paste error?